<?xml version="1.0"?>

<launch>
  <arg name="device_ip"       default="192.168.1.31" />
  <arg name="camera_id"       default="SwissRanger" />
  <arg name="camera_info_url" default="" />

  <node pkg="swissranger_camera" type="swissranger_camera" name="$(arg camera_id)" output="screen" respawn="false">
    <param name="auto_exposure"   value="1" />
    <param name="ether_addr"      value="$(arg device_ip)" />
    <param name="camera_name"     value="$(arg camera_id)" />
    <param name="camera_info_url" value="$(arg camera_info_url)" />
    <!--param name="integration_time" value="40" /-->   
  </node>
  
  <!-- ROS visualizer -->
  <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find swissranger_camera)/cfg/swissranger.rviz" /-->  

</launch>
